save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationContestsMenu.kt

```diff
@@ -137,30 +217,22 @@ private fun organizationContestsMenu() = FC<OrganizationContestsMenuProps> { pro
             onClick = {
                 setIsContestCreationModalOpen(true)
             }
-            +"+ Create contest"
+            +"Create contest"
         }
     }
     div {
         className = ClassName("mb-2")
         contestsTable {
             getData = { _, _ ->
-                val response = get(
-                    url = "$apiUrl/contests/by-organization?organizationName=${props.organizationName}",
-                    headers = Headers().also {
-                        it.set("Accept", "application/json")
-                    },
-                    loadingHandler = ::loadingHandler,
-                )
-                if (response.ok) {
-                    response.unsafeMap {
-                        it.decodeFromJsonString<List<ContestDto>>()
-                    }
-                        .toTypedArray()
-                } else {
-                    emptyArray()
-                }
+                contests.toTypedArray()
             }
             isContestCreated = isToUpdateTable
+            deleteContest = { contest ->
+                setDeletingContest(contest)
+                setConfirmLabel("")
+                setConfirmMessage("Are you sure you want to delete this contest?")
```

Comment:
why we use `state` for label and message?

Reply:
We do this throughout the project to use the same window for different occasions. I also do not think that it needs to be refactored within the framework of this task.

Reply:
no need to refactor `the framework`: just do not use here state for your modal window
Instead of:
```
    displayModal(
        isDeleteContestModalOpen,
        confirmLabel,
        confirmMessage,
        mediumTransparentModalStyle,
        { setDeleteContestModalOpen(false) }
    ) {
        buttonBuilder("Ok") {
            deleteContestFun()
            setDeleteContestModalOpen(false)
        }
        buttonBuilder("Cancel", "secondary") {
            setDeleteContestModalOpen(false)
        }
    }
```
we can use
```
```
    displayModal(
        isDeleteContestModalOpen,
        "",
        "Are you sure you want to delete this contest?",
        mediumTransparentModalStyle,
        { setDeleteContestModalOpen(false) }
    ) {
        buttonBuilder("Ok") {
            deleteContestFun()
            setDeleteContestModalOpen(false)
        }
        buttonBuilder("Cancel", "secondary") {
            setDeleteContestModalOpen(false)
        }
    }
```
```

Reply:
I agree. But if in the future we want to add another window here for confirmation, we will create a new object instead of reusing this one.